metaci/build/flows.py

```diff
@@ -1,17 +1,16 @@
-from cumulusci.core.flows import BaseFlow
+from cumulusci.core.flowrunner import FlowCallback
 from django.utils import timezone
 
 from .models import FlowTask
 
 
-class MetaCIFlow(BaseFlow):
-    """ An implementation of BaseFlow that logs task execution to the database. """
+class MetaCIFlowCallback(FlowCallback):
+    """ An implementation of FlowCallback that logs task execution to the database. """
 
-    def __init__(self, *args, **kwargs):
-        self.buildflow_id = None
-        super(MetaCIFlow, self).__init__(*args, **kwargs)
+    def __init__(self, buildflow_id):
```

Comment:
can just use the default constructor and the ctx variable?

Reply:
I think this makes it more readable by making it obvious what the context _is_.

Reply:
okay sure. can you change the cci branch then to not have the default constructor/ctx member?

Reply:
Done